qmail-remote.c

```diff
@@ -325,6 +417,9 @@ int main(int argc, char **argv)
   if (chdir(auto_qmail) == -1) temp_chdir();
   getcontrols();
  
+#ifdef SMTPUTF8
+  enable_utf8 = env_get("UTF8");
+#endif
```

Comment:
Same as previous. Regarding the environment variable, I'm familiar with setting env vars to control my locale settings, but is this an idiomatic way for a sysadmin to control mail server config? Assuming we're built with EAI, I'm wondering whether EAI should be enabled/disabled systemwide, per domain (incoming and outgoing), or some other level of granularity?

Reply:
And even if all of this is fine this variable name is a bit too generic.

Reply:
Should SMTPUTF8 be ok instead of just UTF8. We can have a control file to have the setting per domain. We could add a line in getcontrols() to read a control file.